lms/migrations/versions/f36f9a0aadf4_add_oauth2_token_table.py

```diff
@@ -0,0 +1,40 @@
+"""
+Add oauth2_token table.
+
+Revision ID: f36f9a0aadf4
+Revises: 7000bb8673c5
+Create Date: 2019-05-15 13:26:41.905098
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+revision = "f36f9a0aadf4"
+down_revision = "7000bb8673c5"
+
+
+def upgrade():
+    op.create_table(
+        "oauth2_token",
```

Comment:
Some tables in the DB have plural names and others have singular names. What is the desired convention for new code?

Reply:
Singular is what h does, so going with that